src/sentry/features/temporary.py

```diff
@@ -191,6 +191,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:issue-views-page-filter", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
     manager.add("organizations:large-debug-files", OrganizationFeature, FeatureHandlerStrategy.INTERNAL, api_expose=False)
     manager.add("organizations:metric-issue-poc", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+    manager.add("organizations:issue-open-periods", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=False)
```

Comment:
should this be `api_expose=True`? i seem to remember someone running into an issue with flagpole with it being set to false

Reply:
`api_expose` is usually only set to `True` if we want to use the flag on the frontend - there's a few examples of flagpole configs with it set to `False`.  The handler strategy _should_ take care of it